client/main.js

```diff
@@ -0,0 +1,24 @@
+import { AppContainer } from 'react-hot-loader'; // required
```

Comment:
I really prefer that we have the liberty to modify the client.js without concern with the HMR option-in structures...